include/icubFloatingBaseEstimatorV1.h

```diff
@@ -0,0 +1,632 @@
+/*
+################################################################################
+#                                                                              #
+# Copyright (C) 2018 Fondazione Istituto Italiano di Tecnologia (IIT)          #
+# All Rights Reserved.                                                         #
```

Comment:
Please use the usual license LGPL2+ , see https://github.com/robotology/blockfactory for an example of nice C++ license headers.

Reply:
Addressed in https://github.com/robotology/whole-body-estimators/pull/1/commits/714c6ccc684abcde00050378b2ddb09d6054ddd9